tokio-macros/src/entry.rs

```diff
@@ -127,21 +127,20 @@ impl Configuration {
 
     fn build(&self) -> Result<FinalConfig, syn::Error> {
         let flavor = self.flavor.unwrap_or(self.default_flavor);
-        use RuntimeFlavor::*;
 
         let worker_threads = match (flavor, self.worker_threads) {
-            (CurrentThread, Some((_, worker_threads_span))) => {
+            (RuntimeFlavor::CurrentThread, Some((_, worker_threads_span))) => {
```

Comment:
I don't mind dropping the import for this case, but I don't want to drop it for `Poll`, `Result`, `Ok`, or `Ordering`. That's too much noise.

Reply:
> I don't mind dropping the import for this case, but I don't want to drop it for `Poll`, `Result`, `Ok`, or `Ordering`. That's too much noise.

Some kinds of enum of them, such as `Result`, `Ordering` might not be changed in the future, unless the std of rust make some broken changes. I reserve my opinion on these enums. But I believe some enums which is defiend in internal code of tokio might change in the future.

We can use `use MyEnum as E` to reduce the noise.

Reply:
I have made some changes, allowing the wildcard in Result and Ordering for reduce the noise. And keep the other changes.